virttest/http_server.py

```diff
@@ -1,13 +1,17 @@
 import os
 import posixpath
-import urlparse
-import urllib
 import logging
-import BaseHTTPServer
-import SimpleHTTPServer
+try:
+    from urllib.parse import unquote, urlparse
+    from http.server import HTTPServer, SimpleHTTPRequestHandler
+except ImportError:
+    from urllib import unquote
+    from urlparse import urlparse
+    from BaseHTTPServer import HTTPServer
+    from SimpleHTTPServer import SimpleHTTPRequestHandler
```

Comment:
It should be separated?
try:
from urllib.parse import unquote, urlparse
except ...

try:
    from http.server import HTTPServer, SimpleHTTPRequestHandler
except ...